### PR TITLE
ci: add codefresh workflows #138

### LINKED ISSUE #138: Add codefresh workflow templates to repo
Add codefresh workflow templates to repo, to track changes in CI/CD

### PR DESCRIPTION
**Pull request by cynful**
#### Description
* add codefresh workflows to repo, to track changes

#### This PR fixes the following issues

Fixes #138

### CODE REVIEW

File: .codefresh/honeydipper-sphinx.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - DOCDST=source
+    commands:
+      - '/opt/honeydipper/drivers/builtin/honeydipper docgen'
+
+  git_commit:
+    title: 'git commit'
+    image: alpine/git:latest
+    commands:
+      - mkdir -p ~/.ssh
+      - echo -n "${SSH_KEY}" | base64 -d > ~/.ssh/id_rsa
```
Comment (huydinhle) [inline .codefresh/honeydipper-sphinx.yml]: Is it possible to use the git provider function from codefresh instead of going this route?.

Comment (wyardley) [reply .codefresh/honeydipper-sphinx.yml]: Might also be worth just doing it in Circle?

**Comment by Charles546** [comment]
I will merge this once it is refreshed with the latest yaml from codefresh.io.  I am kind of not motivated to move to circle and anything else at this stage.  We will revisit this in the future.

**Comment by cynful** [comment]
🤦‍♀️ dont look, its ugly

**Pull request merged**